ledger/src/proofs/transaction.rs

```diff
@@ -559,50 +559,15 @@ impl PlonkVerificationKeyEvals<Fp> {
         CircuitPlonkVerificationKeyEvals::<Fp> {
             sigma: sigma.each_ref().map(cvar),
             coefficients: coefficients.each_ref().map(cvar),
-            generic: cvar(&generic),
-            psm: cvar(&psm),
-            complete_add: cvar(&complete_add),
-            mul: cvar(&mul),
-            emul: cvar(&emul),
-            endomul_scalar: cvar(&endomul_scalar),
+            generic: cvar(generic),
+            psm: cvar(psm),
+            complete_add: cvar(complete_add),
+            mul: cvar(mul),
+            emul: cvar(emul),
+            endomul_scalar: cvar(endomul_scalar),
         }
     }
 
-    /// For debugging
-    fn to_string(&self) -> String {
```

Comment:
This has been moved into a `impl std::fmt::Display`, could you just rename `to_string` to `to_dbg_string` instead ?
And rename `from_string` just below to `from_dbg_string`